src/student_success_tool/generation/pdp/raw_course.py

```diff
@@ -8,7 +8,7 @@
 
 class Provider(BaseProvider):
     def raw_course_record(
-        self, cohort_record: t.Optional[dict] = None, normalize_col_names: bool = False
+        self, cohort_record: t.Optional[dict] = None, normalize_col_names: bool = False, institution_id: int = 12345
```

Comment:
I don't feel super strongly about this, but following existing patterns, it seems like we'd want to pull `institution_id` from the `cohort_record` if provided or otherwise generate a random one, rather than passing cohort record && a specified institution id. In this case, I think you'd just leave all the below code as it was, and just drop this extra arg?

Reply:
Oh ya that seems way better. I was missing some context that you added with your comment.